addons/point_of_sale/static/src/js/models.js

```diff
@@ -806,6 +806,30 @@ exports.PosModel = Backbone.Model.extend({
         });
     },
 
+    // load the missing partner for an order
+    load_missing_partners: function(order){
```

Comment:
Two things: 
- there's already a function that has the same purpose called `_loadMissingPartners` (we're slowly trying to get one naming style in this whole application) so you can maybe try to use this one and adapt the code
- There's no point in implementing a new method in the base module if it's only used in an overridden module, it's better to put it in the module where it's needed.